crates/router/src/connector/stripe/transformers.rs

```diff
@@ -3747,3 +3750,600 @@ mod test_validate_shipping_address_against_payment_method {
+                            .external_account_account_holder_type
+                            .get_required_value("external_account_account_holder_type")
+                            .change_context(errors::ConnectorError::MissingRequiredField {
+                                field_name: "external_account_account_holder_type",
+                            })?,
+                        external_account_account_number: bank_details.bank_account_number,
+                        external_account_routing_number: bank_details.bank_routing_number,
+                    }))
+                }
+                api_models::payouts::Bank::Bacs(_) => Err(errors::ConnectorError::NotSupported {
+                    message: "BACS payouts are not supported".to_string(),
+                    connector: "stripe",
+                }
+                .into()),
+                api_models::payouts::Bank::Sepa(_) => Err(errors::ConnectorError::NotSupported {
+                    message: "SEPA payouts are not supported".to_string(),
+                    connector: "stripe",
+                }
+                .into()),
+            },
+        }
+    }
+}
+
+// Recipient account's creation response
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<types::PayoutsResponseRouterData<F, StripeConnectRecipientAccountCreateResponse>>
+    for types::PayoutsRouterData<F>
+{
+    type Error = Error;
+    fn try_from(
+        item: types::PayoutsResponseRouterData<F, StripeConnectRecipientAccountCreateResponse>,
+    ) -> Result<Self, Self::Error> {
+        let response: StripeConnectRecipientAccountCreateResponse = item.response;
+
+        match response {
+            StripeConnectRecipientAccountCreateResponse::Bank(bank_response) => Ok(Self {
+                response: Ok(types::PayoutsResponseData {
+                    status: Some(enums::PayoutStatus::RequiresCreation),
+                    connector_payout_id: bank_response.id,
+                    payout_eligible: None,
+                }),
+                ..item.data
+            }),
+            StripeConnectRecipientAccountCreateResponse::Card(card_response) => Ok(Self {
+                response: Ok(types::PayoutsResponseData {
+                    status: Some(enums::PayoutStatus::RequiresCreation),
+                    connector_payout_id: card_response.id,
+                    payout_eligible: None,
+                }),
+                ..item.data
+            }),
+        }
+    }
+}
+
+#[cfg(feature = "payouts")]
+impl From<StripeConnectStatus> for enums::PayoutStatus {
+    fn from(stripe_connect_status: StripeConnectStatus) -> Self {
+        match stripe_connect_status {
+            StripeConnectStatus::Succeeded => Self::Success,
+            StripeConnectStatus::Failed => Self::Failed,
+            StripeConnectStatus::Canceled => Self::Cancelled,
+            StripeConnectStatus::Pending
+            | StripeConnectStatus::Processing
+            | StripeConnectStatus::Consumed => Self::Pending,
```

Comment:
What is Consumed, Pls add a one line comment

Reply:
Some of the fields in status are not required for payouts, have updated the same.